clickhouse_search/search.py

```diff
@@ -38,12 +38,11 @@
 )
 
 CLINVAR_FIELDS = OrderedDict({
-    f'key__clinvar__{field.name}': (field.db_column or field.name, field.clone())
+    f'key__clinvar__{field.name}': (field.db_column or field.name, field)
     for field in Clinvar._meta.local_fields if field.name not in CORE_ENTRIES_FIELDS
 })
 
 GENOTYPE_FIELDS = OrderedDict({
-    'project_guid': ('projectGuid', models.StringField()),
```

Comment:
this was never in the genotypes response in the first place, I added it without really thinking